products.js

```diff
@@ -111,27 +197,27 @@ function tunedTitleFnodes(nodeToCssMap) {
     return tuningRoutine;
 }
 
-function tunedPriceFnodes(nodeToCssMap) {
+function tunedPriceFnodes(nodeToCssMap, coeffDollarSign = 1, coeffCSS = 1, coeffSavings = 0.5) {
 
     function hasDollarSign(fnode){
       if(fnode.element.childNodes[0] && fnode.element.childNodes[0].nodeValue && fnode.element.childNodes[0].nodeValue.includes('$')){
-        return 2;
+        return 2 * coeffDollarSign;
       }
       return 1;
     }
 
     function tagHasGoodCss(fnode){
       if(fnode.element.id.match(/(price|sale|deal)/i) || fnode.element.classList.contains(/(price|sale|deal)/i) || fnode.element.itemprop && fnode.element.itemprop.match(/price/i) ||
           fnode.element.classList[0] && fnode.element.classList[0].match(/(price|sale|deal)/i)){
-        return 2;
+        return 2 * coeffCSS;
       }
       return 1;
     }
 
     function notSavingsAmount(fnode){
       const css = nodeToCssMap.get(fnode.element);
       if(css.strikethrough === 'line-through'){
-        return 0;
+        return coeffSavings;
```

Comment:
I see this optimized out to 0.5. I wonder if it really does score better than if you make it 0.